File: backend/java/api-stream/src/src/main/java/org/deus/src/responses/grpc/TokenValidationResponse.java
package org.deus.src.responses.grpc;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.deus.src.dtos.fromModels.UserDTO;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TokenValidationResponse {
    private String tokenStatus;
    private UserDTO userDTO;
}

File: backend/websocket/src/src/responses/token.validation.response.ts
import { UserDto } from '../dtos/user.dto';

export class TokenValidationResponse {
  tokenStatus: string;
  userDTO: UserDto | null;

  public constructor(tokenStatus: string, userDTO: UserDto | null) {
    this.tokenStatus = tokenStatus;
    this.userDTO = userDTO;
  }
}
